.teamcity/builds/apiReferences/kotlinx/io/KotlinxIOBuildApiReference.kt

```diff
@@ -29,13 +29,23 @@ object KotlinxIOBuildApiReference : BuildType({
     }
 
     steps {
-        scriptDropSnapshot {}
+        scriptDropSnapshot {
+            enabled = false
+        }
         scriptDokkaVersionSync {
             enabled = false // TODO readme, and disable this step for other libs too
         }
         buildDokkaHTML {
             enabled = false // TODO add useGradlewWrapper = true
         }
+        script {
+            name = "Drop snapshot"
+            scriptContent = """
+                #!/bin/bash
+                set -e -u
+                sed -i -E "s/^(version=.+)-SNAPSHOT${'$'}/\1/gI" ./gradle.properties
+            """.trimIndent()
```

Comment:
Could you please check, maybe it works via PdeployVersion.
Check out how it is done for metadata:
https://github.com/JetBrains/kotlin-web-site/blob/master/.teamcity/builds/apiReferences/kotlinx/metadataJvm/KotlinxMetadataJvmBuildApiReference.kt#L46

Reply:
Mmm... but it use `KOTLIN_RELEASE_TAG` as i see there is no same version in `kotlinx.io`.
<img width="359" alt="Screenshot 2024-09-19 at 13 15 56" src="https://github.com/user-attachments/assets/98183120-6cd4-48b2-8789-8064d220cd7a">

Reply:
Try KOTLINX_IO_RELEASE_TAG

Reply:
Can you explain me again why we use variable for header?

Reply:
Do you mean why I suggested to use PDeployVersion instead of the script?